### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by scottwn**

### CODE REVIEW

**Comment by scottwn** [comment]
fixes AgnostiqHQ/covalent-staging#295

File: .github/ISSUE_TEMPLATE/FEATURE_REQUEST.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  - type: textarea
+    attributes:
+      label: What should we add?
+      description: A clear and concise description of what you want to happen. If this is related to a problem, describe the problem you're encountering.
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Describe alternatives you've considered
```
Comment (FyzHsn) [inline .github/ISSUE_TEMPLATE/FEATURE_REQUEST.yaml:19]: (non-blocking) Perhaps "What alternatives have you considered?" or add a period at the end of "Describe alternatives ..."

**Review by FyzHsn — APPROVED** [review]
One non-blocking comment, but otherwise good to go!

**Review by FyzHsn — APPROVED** [review]

**Review by FyzHsn — APPROVED** [review]
Looks great!

**Pull request merged**